pkg/trainer/replicas.go

```diff
@@ -139,6 +167,14 @@ func (s *TFReplicaSet) Create() error {
 			return err
 		}
 
+		if s.Spec.IsDefaultPS {
+			// grpc_tensorflow_server.py requires the clusterSpec to be passed as argument in a specific format.
+			// We do the appropriate transformations here
+			cs := transformClusterSpecForDefaultPS(s.Job.ClusterSpec())
+			s.Spec.Template.Spec.Containers[0].Command = []string{"python", "/ps-server/grpc_tensorflow_server.py"}
+			s.Spec.Template.Spec.Containers[0].Args = []string{"--cluster_spec", cs, "--job_name", "ps", "--task_id", fmt.Sprintf("%v", index)}
```

Comment:
Why did you split it into command and args rather than just using command?

Reply:
No good reason, I will change this.